README.md

```diff
@@ -32,9 +32,9 @@ The scanner supports extracting and using the following types of credentials:
 * OAuth2 Refresh Token with cloud-platform scope granted;
 * GCP service account key in JSON format.
 
-The scanner does not rely on any third-party tool (e.g. gcloud). Thus, it can be compiled as a standalone tool and be used on a machine with no GCP SDK installed (e.g. Kubernetes pod). However, please keep in mind that the only OS that is currently supported is Linux. 
+The scanner does not rely on any third-party tool (e.g., gcloud). Thus, it can be compiled as a standalone tool and used on a machine with no GCP SDK installed (e.g., a Kubernetes pod). However, please keep in mind that the only OS that is currently supported is Linux. 
```

Comment:
No commas after e.g.

Reply:
@ggold7046 please resolve this one so we can merge the PR.